src/main/java/uk/gov/cshr/controller/UserController.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.cshr.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.beans.factory.annotation.Value;
+import uk.gov.cshr.service.security.IdentityService;
+
+import javax.servlet.http.HttpServletRequest;
+import java.security.Principal;
+
+@Slf4j
```

Comment:
There is no logging in the code so @Slf4 is not required. Please remove it.

Reply:
Code updated.